### ISSUE TITLE
Adding new loss component

### ISSUE THREAD

**Issue by chandagrover** [hywang66/mmist#6]
Dear Team, 

Thanks again for the great work!!!

Please tell how can I add another loss component. I tried adding new loss component in optimize_parameters() in adaattn_model.py.
 loss = self.loss_content + self.loss_global + self.loss_local  + <new_loss_component>

But, I am not able to see the loss component effects. What changes do need to make the code? I was thinking while we run apply_style_reps.py, the code is using some pre-trained model for the stylization task.

 Please help add a new loss component.
